### PR TITLE
Backport 1.5: Add router svcacct cluster-reader role

### PR DESCRIPTION
**Pull request by mtnbikenc**
Backports #3650

### CODE REVIEW

File: roles/openshift_hosted/tasks/router/router.yml [hunk 1]
```diff
@@ -37,6 +37,15 @@
     resource_name: hostnetwork
   with_items: "{{ openshift_hosted_routers }}"
 
+- name: Set additional permissions for router service account
+  oc_adm_policy_user:
+    user: "system:serviceaccount:{{ item.namespace }}:{{ item.serviceaccount }}"
+    namespace: "{{ item.namespace }}"
+    resource_kind: cluster-role
+    resource_name: cluster-reader
```
Comment (smarterclayton) [inline roles/openshift_hosted/tasks/router/router.yml]: Why is the router a cluster reader?

**Comment by sdodson** [comment]
I'm going to go ahead and pull this in, we're following the documentation. Once we reach consensus on a better implementation we can update both master and release-1.5.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by mtnbikenc** [comment]
aos-ci-test

**Pull request merged**